README.md

```diff
@@ -73,7 +74,47 @@ TBD
 
 ### Llama.cpp
 
-TBD
+**Step 1**. Build Docker image
+
+```bash
+docker build -t llm-perf-llama-cpp:v0.1 -f Dockerfile.cu121.llama_cpp .
+```
+
+**Step 2**. Download the quantized GGML models and run Llama2 via llama.cpp.
+
+To obtain the quantized GGML model, it is recommended to download it via HuggingFace using
+the comamnd below:
+
+```bash
+wget https://huggingface.co/TheBloke/Llama-2-7B-GGML/resolve/main/llama-2-7b.ggmlv3.q4_K_M.bin
+wget https://huggingface.co/TheBloke/Llama-2-13B-GGML/resolve/main/llama-2-13b.ggmlv3.q4_K_M.bin
+```
+
+```bash
+PORT=41514
+GGML_BINS=/PATH/TO/GGML_BINS/  # Replace it with path to HuggingFace models
+
+docker run                  \
+  -d -P                     \
+  --gpus all                \
+  -h llm-perf               \
+  --name llm-perf-llama-cpp \
+  -p $PORT:22               \
+  -v $GGML_BINS:/ggml_bins  \
+  llm-perf-llama-cpp:v0.1
+
+# Password is: llm_perf
+ssh root@0.0.0.0 -p $PORT
+```
+
+**Step 3.** Run the CLI tool to see the performance numbers:
+
+Log in to the docker container we created using the comamnd below:
+
+```bash
+cd $LLAMA_CPP_HOME
+./build/bin/main -m /ggml_bins/llama-2-7b.ggmlv3.q4_K_M.bin -p "Please generate a very long story about wizard and technology, at least two thousand words" -n 1920 -ngl 999 --ignore-eos |& tee llama_cpp_result.log
```

Comment:
Note that we are using a pretty short prompt here, may be replaced with a standard length promot if necessary. In MLC-LLM we are using randomly generated input with given input length.